src/hooks/userAnalyticHelpers.js

```diff
@@ -0,0 +1,7 @@
+// Pushing data to segment analytics
```

Comment:
technically this isn't a hook. heh. 
You could refactor it, and have the segment analytics initialize through through a `useEffect(()=>{},[])` and maintain the state for the `analytics` object. probably a better pattern imo.

Reply:
Okay! we can sync to hash out precisely how to get it done, happy to iron it out. ty for the review!